spec/controllers/api/v1/operators_controller_spec.rb

```diff
@@ -82,18 +82,18 @@
 
   describe 'GET show' do
     context 'as JSON' do
-      it 'returns operators by OnestopId' do
+      it 'returns operators by Onestop ID' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.